drivers/interrupt_controller/intc_gicv3.c

```diff
@@ -333,7 +338,12 @@ SYS_INIT(arm_gic_init, PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
 #ifdef CONFIG_SMP
 void arm_gic_secondary_init(void)
```

Comment:
Not sure, can we combine `arm_gic_init` and `arm_gic_secondary_init` together, since they are almost the same?
Something like:
int arm_gic_init(const struct device *unused)
{
    gicv3_dist_init();
    __arm_gic_init(); //whatever
}

void arm_gic_secondary_init(void)
{
     __arm_gic_init();
}

Reply:
Done, thanks.